### DISCUSSION TITLE
globalSearchKeyBindings with forward slash does not work

### DISCUSSION THREAD

**Discussion by getrealwealth** [filamentphp/filament#13019] [Help]
### Package

Panel builder

### Package Version

v3.2.84

### How can we help you?

I'm trying to do forward slash on Global search with

```
return $panel
    ->globalSearchKeyBindings(['/'])
```
But it does not work. Every other key, including backward slash, works but not forward slash.

**Comment by blazerunner44** [comment]
Same issue here.

**Comment by sparksp** [comment]
I have done some digging and submitted an issue and pull request upstream in https://github.com/danharrin/alpine-mousetrap/pull/10 that should help us.
